tests/conftest.py

```diff
@@ -11,6 +11,14 @@
     import sys
     sys.stdout = sys.stderr
 
+def pytest_addoption(parser):
+    parser.addoption('--gpu_available', action='store_true', dest="gpu_available",
+                 default=False, help="enable GPU tests")
```

Comment:
This function is overwritten by the function `pytest_addoption` below. Please move the `addoption` statement to inside the existing function

Reply:
done thanks